modules/nw-metallb-example-bgppeer.adoc

```diff
@@ -82,3 +83,29 @@ spec:
   peerASN: 64500
   myASN: 64500
 ----
+
+[id="nw-metallb-example-unnumbered-bgp-peering_{context}"]
+== Example: Specify BGP peers for unnumbered BGP peering
+
+include::snippets/technology-preview.adoc[]
+
+To configure unnumbered BGP peering, specify the interface in the `spec.interface` field.
+
+[source,yaml]
+----
+apiVersion: metallb.io/v1beta2
+kind: BGPPeer
+metadata:
+  name: peer-unnumber
+  namespace: metallb-system
+spec:
+  myASN: 64512
+  ASN: 645000
+  interface: net0
+----
+[NOTE]
+====
+To use the `interface` field, there must be a direct, point-to-point connection between the two BGP peers, and you must enable IPv6 RAs (Router Advertisements).
```

Comment:
Again here its not a traditional p2p but it needs to be layer 2. The most important point is that there can only be one BGP connection per interface.